R/use_tic.R

```diff
@@ -346,6 +371,7 @@ ci_menu <- function(choices, title) {
     travis = "Travis CI",
     circle = "Circle CI",
     appveyor = "AppVeyor CI",
+    ghactions = "Github Actions",
```

Comment:
Should we be consistent with the trademark here?

```suggestion
    ghactions = "GitHub Actions",
```

Reply:
Ok, then I'll do it pkg wide.